packages/picker/test/picker.test.ts

```diff
@@ -382,7 +363,7 @@ describe('Dropdown', () => {
         );
     });
     it.only('opens on ArrowDown', async () => {
-        const el = await dropdownFixture();
+        const el = await pickerFixture();
```

Comment:
This wasn't you, but while we're here, can you correct the two uses of `it.only` here and above?